src/browser/components/SkillItem.react.js

```diff
@@ -16,46 +15,23 @@ export default class SkillItem extends Component {
   handleChange(level, e) {
     const {actions: {saveSkill}} = this.props;
     saveSkill(this.props.data, level);
-    this.props.resetErrorStatus();
   }
 
-  setHintVisibility(visible) {
-    this.setState({hintDisplayed: visible});
-  }
-
-  handleMouseEnter = () => this.setHintVisibility(true);
-  handleMouseLeave = () => this.setHintVisibility(false);
-
   render() {
-    let hint;
-    const cb = {
-      onMouseEnter: this.handleMouseEnter,
-      onMouseLeave: this.handleMouseLeave
-    };
-
-    if (this.state.hintDisplayed) {
-      hint = (<div {...cb} className="hint">{this.props.data.name}</div>);
+    const starsLabels = [];
+    for (let i = 5; i >= 1; i--) {
```

Comment:
- `5` could be a constant. That way it would be more descriptive. 
- Standard way for doing for loops is to start from 0 and increament. Is there any reason that you choosed to make it different?